integration-tests/gradle/src/main/resources/basic-composite-build-extension-project/extensions/example-extension/deployment/src/main/java/org/acme/example/extension/deployment/QuarkusExampleProcessor.java

```diff
@@ -34,13 +31,4 @@ void addLibABean(BuildProducer<AdditionalBeanBuildItem> additionalBeans) {
 				.build());
 	}
 
-
-	@BuildStep
-	@Record(ExecutionTime.STATIC_INIT)
-	EnabledBuildItem addLibABean(
-			final ExampleRecorder exampleRecorder) {
-		return new EnabledBuildItem(exampleRecorder.create().getValue());
-
-	}
-
```

Comment:
@tomas1885 would you like to review these changes?

Reply:
I think the recorder was there just to make sure that it's that nothing is broken between the runtime and deployment modules. It might be redundant so maybe it's ok to remove it, as long as we're sure the current test cases covers this.